randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3094,10 +3102,10 @@ Extra - asset_id: collision_camera_038
   * Open Passage to Thermal Device Room North/Dock to Lava Button West (Bottom)
   > Tunnel to Thermal Device Room North
       Any of the following:
+          Gravity Suit or After Cataris - Lower Lava Button
           All of the following:
-              Gravity Suit or After Cataris - Lower Lava Button
-              Space Jump or Speed Booster or Simple IBJ
-          Space Jump and Lava Damage ≥ 500
+              Heat/Cold Runs (Advanced) and Lava Damage ≥ 500
+              Varia Suit or Heat Damage ≥ 500
```

Comment:
I assume you were able to walljump out of the lava then?

Reply:
Correct